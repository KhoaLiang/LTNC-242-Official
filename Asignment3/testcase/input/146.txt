INSERT x string
ASSIGN x 5