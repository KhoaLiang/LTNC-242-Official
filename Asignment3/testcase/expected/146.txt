TypeMismatch: ASSIGN x 5